include/openspace/engine/globalscallbacks.h

```diff
@@ -42,12 +42,12 @@ inline std::vector<std::function<void()>>* postSyncPreDraw;
 inline std::vector<std::function<void()>>* render;
 inline std::vector<std::function<void()>>* draw2D;
 inline std::vector<std::function<void()>>* postDraw;
-inline std::vector<std::function<bool(Key, KeyModifier, KeyAction)>>* keyboard;
-inline std::vector<std::function<bool(unsigned int, KeyModifier)>>* character;
-inline std::vector<std::function<bool(MouseButton, MouseAction, KeyModifier)>>*
+inline std::vector<std::function<bool(Key, KeyModifier, KeyAction, bool)>>* keyboard;
```

Comment:
Should we declare a new `Boolean`  `GuiWindow` to use here instead?

Reply:
I like that idea. Would make the meaning of the bool more apparent 

Like this: `using IsGuiWindow = bool; ` ? In this file or somewhere else?

Reply:
I guess we might also want to use it in the cpp file, and in the 

```
using F = std::function<bool (unsigned int, KeyModifier, bool)>;
```
declarations in the openspaceengine.cpp file

Reply:
In ghoul/misc/boolean.h   there is a `Boolean` define that does that type safe.  So:

`Boolean(IsGuiWindow);` in this file and then use `IsGuiWindow::Yes`  and `IsGuiWindow::No` when using it.  It autoconverts into a bool, too.

Reply:
Cool, thanks!